### PR TITLE
Update new send icon

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to update the send icon in the wallet view with the new icon

### Before

Screenshot 2023-04-03 at 4 25 24 PM

### After

Screenshot 2023-04-03 at 4 22 48 PM

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: app/images/icons/arrow-2-up-right.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M6.4375 5C6.4375 4.44772 6.88522 4 7.4375 4H18C18.5523 4 19 4.44772 19 5V15.5625C19 16.1148 18.5523 16.5625 18 16.5625C17.4477 16.5625 17 16.1148 17 15.5625V7.41421L5.70711 18.7071C5.31658 19.0976 4.68342 19.0976 4.29289 18.7071C3.90237 18.3166 3.90237 17.6834 4.29289 17.2929L15.5858 6H7.4375C6.88522 6 6.4375 5.55228 6.4375 5Z" fill="#24272A"/>
+</svg>
```
Comment (georgewrmarshall) [inline app/images/icons/arrow-2-up-right.svg:3]: This icon doesn't look like it has been optimized? Here is a video on how to do that https://www.notion.so/Export-icons-as-optimized-svgs-for-code-daf05f66cf2e44fc8306d48a3cbbb930?pvs=4

**Review by georgewrmarshall — COMMENTED** [review]
Looking good. Need to optimize the icon svg

**Review by vthomas13 — DISMISSED** [review]

**Review by DDDDDanica — DISMISSED** [review]

File: app/images/icons/arrow-2-up-right.svg (hunk 1 above)
Comment (NidhiKJha) [reply app/images/icons/arrow-2-up-right.svg:3]: Good shout. Optimised it here https://github.com/MetaMask/metamask-extension/pull/18411/commits/de88b833aa1b439f638825f9e33036267adca7ba

Comment (DDDDDanica) [reply app/images/icons/arrow-2-up-right.svg:3]: wow this is super useful i didn't know you can combine the path, thanks for the tutorial @georgewrmarshall

**Review by DDDDDanica — DISMISSED** [review]

File: app/images/icons/arrow-2-up-right.svg (hunk 1 above)
Comment (georgewrmarshall) [reply app/images/icons/arrow-2-up-right.svg:3]: Nice work!

**Review by georgewrmarshall — COMMENTED** [review]
Great work on optimizing the SVG! Would it be possible to add this to the enum as well? https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/component-library/icon/icon.types.ts#L22

I'll do another sync when I remove the generation of `ICON_NAMES` but can't hurt to add it there in the meantime

**Comment by NidhiKJha** [comment]
> Great work on optimizing the SVG! Would it be possible to add this to the enum as well? https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/component-library/icon/icon.types.ts#L22
> 
> I'll do another sync when I remove the generation of `ICON_NAMES` but can't hurt to add it there in the meantime

Updated here https://github.com/MetaMask/metamask-extension/pull/18411/commits/19b2aeafd7e9765afad5ae41fa2f7921c8ce4a84

**Review by vthomas13 — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]
LGTM! 💯

**Pull request merged**